<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogForCreatingNewEmployee</class>
 <widget class="QDialog" name="DialogForCreatingNewEmployee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resources/resource/img/vigor-300x291.png</normaloff>:/resources/resource/img/vigor-300x291.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="FirstName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Prezime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SecondName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Korisnicko Ime</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="UserName"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QComboBox" name="Position">
          <item>
           <property name="text">
            <string>Administrator</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Komercijalista</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Proizvodnja</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dizajner Lastis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dizajner Tkanje</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Dizajner Stampa</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pakovanje</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Magacioner</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sef Proizvodnje</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sef Smene</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Knjigovodja Fakture</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pristup Sistemu</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="PristupSistemu">
          <item>
           <property name="text">
            <string>Da</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Ne</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogForCreatingNewEmployee</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogForCreatingNewEmployee</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
